### ISSUE TITLE
Can I train this model with my own dataset? Is there any relevant introduction?

### ISSUE THREAD

**Issue by SE-Nickjackson** [kiva12138/CubeMLP#4]
Sorry to bother you, I want to ask that if I can train this model with my own dataset? Is there any relevant introduction?

**Comment by kiva12138** [comment]
Yes, of course. I think CubeMLP can be used for any multimodal tasks because it is designed without any prior knowledge.

If your own dataset incorporate textual modality, the only thing you should do is to create the .pkl files (actually, it's also not necessary if just want to use the model).
If your own dataset has no textual modality, you are supposed to delete the irrelevant codes which are for textual processing.


In summary, CubeMLP expects the inputs as  a Tesnor with shape A*B*C*...*F, and it will compute the affine trasformation on any axis. You can  also define some more complex operations.

Currently there is still no intuitive tutorials in this repo, we will try our best to provide some relevant tutorials in the future.

Thanks very much.
